spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context 'Testing User Modedl' do
+    before(:each) do
+      user1 = User.create(user_name: 'foo', email: 'foo@bar.com')
```

Comment:
The linter is right assigning users1 is useless because you do not use it. In general this before(:each) is not actually being used. Remove before merging